libxmf/XmfMuxer.h

```diff
@@ -0,0 +1,20 @@
+#include <xmf/xmf.h>
+#ifndef XMF_MUXER_H
+#define XMF_MUXER_H
+
+typedef struct xmf_webm_muxer XmfWebMMuxer;
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+	XMF_EXPORT XmfWebMMuxer* XmfWebMMuxer_New();
+
+	XMF_EXPORT int XmfWebMMuxer_Remux(XmfWebMMuxer* muxer, const char* inputPath, const char* outputPath);
```

Comment:
right now Remux doesn't accept any options, it just post-processes a video file to add cues for seeking, but we could envision adding XmfWebMMuxer_* options in the future before calling Remux if need be in the future.

Reply:
Agree, this could come in future patches once we know what we need exactly